### PR TITLE
Add a simple playwright setup

### PR DESCRIPTION
**Pull request by tmeasday**
Issue: https://linear.app/chromaui/issue/SB-623/add-playwright-and-create-simple-test

## What I did

- Added playwright and simple test based on our existing cypress ones
- Added it to CI
- Stored artifacts and test results

### CODE REVIEW

File: code/.gitignore [hunk 1]
```diff
@@ -36,6 +36,12 @@ examples/angular-cli/addon-jest.testresults.json
 junit.xml
 .next
 
+/test-results/
+/playwright-results/
+/playwright-report/
+/playwright/.cache/
```
Comment (ndelangen) [inline code/.gitignore:42]: Should we aim to have these files in `/scripts` instead?

Comment (tmeasday) [reply code/.gitignore:42]: Makes sense. Where should the actual script live @ndelangen ?

**Comment by ndelangen** [comment]
@tmeasday Long term, I think we'll move all npm scripts into dev scripts run from the root.
For now, we can keep the tradition of having npm scripts in the code directory's package.json

**Comment by yannbf** [comment]
Hey @tmeasday this seems good but there is a flake going on and the tests are passing. We should probably find out a way to surface this so we don't keep accepting flake without noticing

**Review by yannbf — APPROVED** [review]

**Comment by tmeasday** [comment]
Merged this into https://github.com/storybookjs/storybook/pull/18886 and going to close in favour of that.

**Pull request closed without merging**